packages/components/asset/README.mdx

```diff
@@ -1,46 +1,43 @@
 ---
 title: 'Asset'
-type: 'component'
-status: 'stable'
 slug: /components/asset/
 github: 'https://github.com/contentful/forma-36/tree/master/packages/components/asset'
 storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-asset--basic'
 typescript: ./Asset.tsx
```

Comment:
There is something wrong with the PropsTable, could you check it please?

![Screenshot 2022-01-17 at 17 31 59](https://user-images.githubusercontent.com/6597467/149807306-f817855b-fa18-4090-8c2d-5f960419513d.png)